prime-angular-frontend/src/app/modules/auth/pages/phsa/phsa.component.ts

```diff
@@ -32,9 +32,9 @@ export class PhsaComponent implements OnInit {
     return this.viewportService.isMobile;
   }
 
-  public loginUsingBCSC() {    
-    // TODO: redirect route to the access code page in the PHSA Labtech module
-    const redirectRoute = PhsaLabtechRoutes.routePath(PhsaLabtechRoutes.EXAMPLE);
+  public loginUsingBCSC() {
+    // redirect route to the access code page in the PHSA Labtech module
```

Comment:
Feels like the comment can be removed since the routing and destination are explicit

Reply:
comments removed